digit-health-services/libraries/health-services-common/src/main/java/org/digit/health/common/data/query/builder/SelectQueryBuilder.java

```diff
@@ -0,0 +1,19 @@
+package org.digit.health.common.data.query.builder;
+
+import java.util.List;
+
+public class SelectQueryBuilder implements GenericQueryBuilder {
+
+    @Override
+    public String build(Object object) {
+        StringBuilder queryStringBuilder = null;
+        try {
+            String tableName = GenericQueryBuilder.getTableName(object.getClass());
+            List<String> whereClauses = GenericQueryBuilder.getFieldsWithCondition(object, QueryFieldChecker.isNotNull);
+            queryStringBuilder = GenericQueryBuilder.generateQuery(GenericQueryBuilder.selectQueryTemplate(tableName), whereClauses);
+        } catch (Exception exception) {
+            System.out.println(exception.getMessage());
```

Comment:
Create exception QueryBuilderException in the query folder and throw it from here by putting exception.getMessage() in the constructor.
`throw new QueryBuilderException(exception.getMessage);`

Reply:
Created QueryBuilderException.